app/client/ui2018/alerts.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * A simple alert component that displays a highlighted message with an info icon.
+ *
+ * Example:
+ *
+ * `alert('This is some important information')`
+ */
+import { dom, DomElementArg, styled } from 'grainjs';
+import { vars } from 'app/client/ui2018/cssVars';
+import { icon } from 'app/client/ui2018/icons';
+
+/**
+ * Creates an alert element with an icon and message.
+ */
+export function alert(...args: DomElementArg[]) {
+  return cssAlert(
+    cssAlertIcon(
+      icon('Info',
+        dom.style('width', '13px'),
+        dom.style('height', '13px'),
+        dom.style('background-color', 'currentColor'),
+      ),
+    ),
+    dom('p', ...args)
+  );
+}
+
+const cssAlert = styled('div', `
```

Comment:
Why not using a `<dialog>` element?

Reply:
It's not a dialog, this is like a [bootstrap alert component](https://getbootstrap.com/docs/5.3/components/alerts/), just a styled normal element that we visually highlight :)